scripts/gasCompare.js

```diff
@@ -0,0 +1,80 @@
+const fs = require("fs");
+const markdown = require('json-to-markdown-table');
+const Commenter = require('../node_modules/circleci-pr-commenter/dist/index');
```

Comment:
Why require like this? Check like 2, should be like that.

Reply:
Yeah, I will correct this.